### PR TITLE
Switch cf api queries to use POST request

### PR DESCRIPTION
**Pull request by algmyr**
Superficial test seemed to work. Hard to check in the cases where is actually matters without painstakingly creating a huge db for testing. Testing something like `;handle unmagic_all` on server should at least verify that the limit has vastly increased.

### CODE REVIEW

File: tle/util/codeforces_api.py [hunk 1]
```diff
@@ -335,14 +334,32 @@ async def problems(*, tags=None, problemset_name=None):
...
     @staticmethod
     async def info(*, handles):
-        chunks = chunkify(handles, MAX_HANDLES_PER_QUERY)
-        if len(chunks) > 1:
-            logger.warning(f'cf.info request with {len(handles)} handles,'
-            f'will be chunkified into {len(chunks)} requests.')
+        chunks = user_info_chunkify(handles)
+        #if len(chunks) > 1:
+        #    logger.warning(f'cf.info request with {len(handles)} handles,'
+        #    f'will be chunkified into {len(chunks)} requests.')
```
Comment (meooow25) [inline tle/util/codeforces_api.py:362]: Don't we want the warning any more? 🤔

File: tle/util/codeforces_api.py [hunk 2]
```diff
@@ -335,14 +334,32 @@ async def problems(*, tags=None, problemset_name=None):
...
+            size = 0
+        chunk.append(handle)
+        size += len(handle) + 1
+    if chunk:
+        yield chunk
 
 class user:
     @staticmethod
     async def info(*, handles):
-        chunks = chunkify(handles, MAX_HANDLES_PER_QUERY)
```
Comment (meooow25) [inline tle/util/codeforces_api.py:342]: `chunkify` import no longer needed.

File: tle/util/codeforces_api.py [hunk 3]
```diff
@@ -249,7 +248,7 @@ async def _query_api(path, params=None):
         logger.info(f'Querying CF API at {url} with {params}')
         # Explicitly state encoding (though aiohttp accepts gzip by default)
         headers = {'Accept-Encoding': 'gzip'}
-        async with _session.get(url, params=params, headers=headers) as resp:
+        async with _session.post(url, params=params, headers=headers) as resp:
```
Comment (meooow25) [inline tle/util/codeforces_api.py:251]: I think we need to change `params` to [`data`](https://docs.aiohttp.org/en/stable/client_quickstart.html#more-complicated-post-requests).

File: tle/util/codeforces_api.py (hunk 1 above)
Comment (algmyr) [reply tle/util/codeforces_api.py:362]: I was meaning to remove is for the sake for keeping chunks as an iterator, but I guess doing list and keeping the check is fine as well.

File: tle/util/codeforces_api.py (hunk 3 above)
Comment (algmyr) [reply tle/util/codeforces_api.py:251]: Curiously it accepts both and seem to treat them more or less equivalently. But yeah, data is the right one for post.

**Review by meooow25 — APPROVED** [review]

**Pull request merged**